Snowflake.Data.Tests/FastMemoryStreamTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+namespace Snowflake.Data.Tests
+{
+    using NUnit.Framework;
+    using Snowflake.Data.Core;
+    using System.Linq;
+
+    [TestFixture]
+    class FastMemoryStreamTest
+    {
+        FastMemoryStream fastMemoryStream;
+
+        [SetUp]
+        public void BeforeTest()
+        {
+            fastMemoryStream = new FastMemoryStream();
+        }
+
+        [Test]
+        [Ignore("FastMemoryStreamTest")]
+        public void FastMemoryStreamTestDone()
+        {
+            // Do nothing;
+        }
+
+        [Test]
+        public void TestDefaultValues()
+        {
+            // FastMemoryStream defaults to size of 0 and buffer of size 256
```

Comment:
is 256 defined somewhere as constant? can we reuse such constant then?

Reply:
There was no constant but I made one for it